scss/index.scss

```diff
@@ -2,25 +2,26 @@
 @import 'variables';
 @import 'colors';
 @import 'mixins';
+@import 'footer';
 
-// body {
-// 	padding-left: 0;
-// 	padding-right: 0;
-// 	font-family: $family_gotham;
-// 	font-size: $font_size_base;
-// }
+body {
```

Comment:
@laurxnemeth I was concerned that making these font changes would make many things weird but I prefer that you pulled off the band-aid!